ci-automation/vendor-testing/qemu.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Test execution script for the qemu vendor image.
+# This script is supposed to run in the SDK container.
+
+function run_testsuite() {
```

Comment:
Where does this function get called? I think the call is missing at the end of this file, or?

Reply:
Yes, this script is still work in progress. I did the generic / specific split very late when writing the initial draft, I don't think there needs to be a wrapper function at all.